packages/console/src/Console.jsx

```diff
@@ -453,6 +457,11 @@ export class Console extends PureComponent {
     });
   }
 
+  updateObjectMap() {
+    const { objectMap } = this.props;
+    this.setState({ objectMap });
```

Comment:
`objectMap` is optional according to `PropTypes`. Either do `{ objectMap: new Map(objectMap) }` here or update `PropTypes` (then drop `new Map()` in the `constructor` too to match `updateObjectMap`)